[OSDOCS-6944] content port what's new update (#62596)
openshift/openshift-docs: rosa_release_notes/rosa-release-notes.adoc

[id="rosa-q3-2023_{context}"]
=== Q3 2023

* **Documentation update:** The Monitoring section in the documentation was expanded and now includes more detailed information to help you conveniently manage your ROSA clusters. For more information, see xref:../monitoring/monitoring-overview.html[Monitoring overview].

* **ROSA regional availability update:** {product-title} (ROSA) is now available in the Hyderabad (`ap-south-2`) region. For more information on region availability, see xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-regions-az_rosa-service-definition[Regions and availability zones].

[id="rosa-q2-2023_{context}"]
=== Q2 2023

> * **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.23[release notes]. For more information about the ROSA CLI (`rosa`,) see xref:../rosa_cli/rosa-get-started-cli.html[Getting started with the rosa CLI].

Reviewer: ```suggestion
* **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.23[release notes]. For more information about the ROSA CLI (`rosa`), see xref:../rosa_cli/rosa-get-started-cli.html[Getting started with the rosa CLI].
```

Reviewer: @aldiazRH do you want the comma inside the parentheses? Personally, I think it should be outside, but I defer to you. That's my only comment here. Otherwise, LGTM to merge.

[id="rosa-q3-2023_{context}"]
=== Q3 2023

> * **Documentation update:** The **Monitoring** section in the documentation has been expanded and now includes more detailed information to help you conveniently manage your ROSA clusters. For more information, see xref:../monitoring/monitoring-overview.html[Monitoring overview].

Reviewer 2: ```suggestion
* **Documentation update:** The Monitoring section in the documentation was expanded and now includes more details to help you conveniently manage your ROSA clusters. For more information, see xref:../monitoring/monitoring-overview.html[Monitoring overview].
```
Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element), no highlighting is needed for the names of documentation sections. If it's standard to use bold to refer to documentation sections in the ROSA docs, feel free to disregard my suggestion. 

I replaced "has been" with "was" to use the simple present tense, per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).
